### PR TITLE
[UserBundle] CS fixes and missing specs

### PR DESCRIPTION
**Pull request by aRn0D**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | refactoring 
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | no
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Sylius/Bundle/UserBundle/spec/OAuth/UserProviderSpec.php [hunk 1]
```diff
@@ -26,145 +26,150 @@
...
+//        $user->setUsername('user')->shouldBeCalled();
+//        $user->setPlainPassword('2ff2dfe363')->shouldBeCalled();
+//        $user->setEnabled(true)->shouldBeCalled();
+//        $user->addOAuthAccount($oauth)->shouldBeCalled();
+//
+//        $userManager->persist($user)->shouldBeCalled();
+//        $userManager->flush()->shouldBeCalled();
+//
+//        $this->loadUserByOAuthUserResponse($response)->shouldReturn($user);
+//    }
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/UserBundle/spec/OAuth/UserProviderSpec.php]: Why is it all commented out?

Comment (aRn0D) [inline src/Sylius/Bundle/UserBundle/spec/OAuth/UserProviderSpec.php]: because I failed !

**Comment by pjedrzejewski** [comment]
@aRn0D The build seems to be failing with a missing class error. :octocat:

File: app/config/config_dev.yml [hunk 2]
```diff
@@ -27,4 +27,4 @@ assetic:
 
 doctrine:
     dbal:
-        dbname: %sylius.database.name%_dev
+        dbname: %sylius.database.name%_test
```
Comment (Shine-neko) [inline app/config/config_dev.yml]: why ?

Comment (aRn0D) [inline app/config/config_dev.yml]: Failed

**Comment by aRn0D** [comment]
@pjedrzejewski I will have a look at midday.

**Comment by aRn0D** [comment]
@pjedrzejewski green with the VariationBundle fixes

**Comment by pjedrzejewski** [comment]
Thank you Arnaud! :)

**Pull request merged**